app/controllers/home_controller.rb

```diff
@@ -1,15 +1,16 @@
 class HomeController < ApplicationController
 
-  def show
-    @user_dashboard = UserDashboard.new(user: current_user)
-    @repo = Repo.new
-    @repos = current_user.repos.where(repo_id: current_repo)
-  end
+  # def show
```

Comment:
@DavidKnott As discussed during our joint review, once the refactor is mature enough and no side effects found, we should clean up the commented out code.